fury/molecular.py

```diff
@@ -0,0 +1,483 @@
+
+    def setMultiTubesForBonds(self, choice):
+        """Set whether multiple tubes will be used to represent multiple bonds.
+
+        Parameters
+        ----------
+        choice : bool
+            If choice is True, multiple bonds (double, triple) will be shown by
+            using multiple tubes.
+            If choice is False, all bonds (single, double, triple) will be
+            shown as single bonds (i.e shown using one tube each).
+        """
+        self.SetUseMultiCylindersForBonds(choice)
+
+    def useMolecularSphereRep(self, colormode):
+        """Set Mapper settings to create a molecular sphere representation.
+
+        Parameters
+        ----------
+        colormode : string
+            Set the colormode for coloring the atoms. Two valid color modes -
+        * 'discrete': each atom is colored using the internal lookup table.
+        * 'single': all atoms of same color.
+        """
+        self.setRenderAtoms(True)
+        self.setRenderBonds(False)
+        self.setAtomicRadiusTypeToVDWRadius()
+        self.setAtomicRadiusScaleFactor(1.0)
+        if colormode == 'discrete':
+            self.setAtomColorModeToDiscrete(True)
+        elif colormode == 'single':
+            self.setAtomColorModeToDiscrete(False)
+
+    def useMolecularBallStickRep(self, colormode, atom_scale_factor,
+                                 bond_thickness, multipleBonds):
+        """Set Mapper settings to create a molecular ball and stick
+        representation.
+
+        Parameters
+        ----------
+        colormode : string
+            Set the colormode for coloring the atoms. Two valid color modes -
+            * 'discrete': each atom is colored using the internal lookup table.
+            * 'single': all atoms of same color.
+        atom_scale_factor : float
+            Scaling factor to be applied to the atoms.
+        bond_thickness : float
+            Used to manipulate the thickness of bonds (i.e. thickness of tubes
+            which are used to render bonds)
+        multipleBonds : string
+            Set whether multiple tubes will be used to represent multiple
+            bonds. Two valid choices -
+            * 'On': multiple bonds (double, triple) will be shown by using
+              multiple tubes.
+            * 'Off': all bonds (single, double, triple) will be shown as single
+              bonds (i.e shown using one tube each).
+        """
+        if self.bonds_data_available:
+            self.setRenderAtoms(True)
+            self.setRenderBonds(True)
+            self.setBondThickness(bond_thickness/10)
+            self.setAtomicRadiusTypeToVDWRadius()
+            self.setAtomicRadiusScaleFactor(atom_scale_factor)
+            if multipleBonds == 'On':
+                self.setMultiTubesForBonds(True)
+            elif multipleBonds == 'Off':
+                self.setMultiTubesForBonds(False)
+        else:
+            print("Inadequate Bonding data")
```

Comment:
For better error handling, should I make a custom exception? 
Something like -
```
class BondingError(Exception):
 pass
```
and then use it by calling 
`raise BondingError("Inadequate Bonding Error!")`

Reply:
Done